src/main/java/com/scriptbasic/interfaces/ScriptBasicKeyWords.java

```diff
@@ -5,11 +5,61 @@
  * date June 26, 2012
  */
 public interface ScriptBasicKeyWords {
-    String[] BASIC_KEYWORDS = new String[]{"for", "end", "next", "let", "if",
-            "endif", "then", "while", "wend", "else", "elseif", "repeat",
-            "until", "not", "false", "true", "and", "or", "div", "use", "from",
-            "as", "to", "step", "method", "is", "rem", "sub", "endsub",
-            "return", "print", "global", "local", "call", "sentence"};
+    public static final String KEYWORD_AND = "and";
```

Comment:
Because this is an `interface` the keywords `public`, `static` and `final` are meaningless. A field in an interface is by default `public static final` and it cannot be anything else.

Please delete these keywords.